Windows build failure in nanoBragg

I'm currently getting a failure while trying to build DIALS on Windows. The end of the build log states this:
```
C:\Users\fcx32934\sw\cctbx-build-test\modules\cctbx_project\simtbx\nanoBragg\nanoBragg_ext.cpp(31): note: while trying to match the argument list '(double)'
C:\Users\fcx32934\sw\cctbx-build-test\modules\cctbx_project\simtbx\nanoBragg\nanoBragg_ext.cpp(31): error C2660: 'boost::python::make_tuple': function does not take 4 arguments
c:\Users\fcx32934\sw\cctbx-build-test\conda_base\Library\include\boost/python/tuple.hpp(62): note: see declaration of 'boost::python::make_tuple'
scons: building terminated because of errors.
```
Curiously, the line in question,
https://github.com/cctbx/cctbx_project/blob/a47b4700fe1473e8296598c9f4a1c2544282a25e/simtbx/nanoBragg/nanoBragg_ext.cpp#L31
was modified 4 years ago in adb332f00e as part of fixes required for Windows builds

--- a/simtbx/nanoBragg/nanoBragg.cpp
+++ b/simtbx/nanoBragg/nanoBragg.cpp
@@ -6,6 +6,8 @@
 namespace simtbx {
 namespace nanoBragg {
 
+using boost::math::isnan;
+
 class encapsulated_twodev
 {
 /* Convert gaussdev from a function to a class.

--- a/simtbx/nanoBragg/nanoBragg_ext.cpp
+++ b/simtbx/nanoBragg/nanoBragg_ext.cpp
@@ -28,7 +28,7 @@ namespace boost_python { namespace {
   testuple()
   {
     double test = NAN;
-    return boost::python::make_tuple(isnan(test),2,3,4);
+    return boost::python::make_tuple(boost::python::isnan(test),2,3,4);
   }
 
   /* getter/setter functions for convenient python-side members

--- a/simtbx/nanoBragg/nanoBragg_ext.cpp
+++ b/simtbx/nanoBragg/nanoBragg_ext.cpp
@@ -28,7 +28,7 @@ namespace boost_python { namespace {
   testuple()
   {
     double test = NAN;
-    return boost::python::make_tuple(boost::python::isnan(test),2,3,4);
+    return boost::python::make_tuple(boost::math::isnan(test),2,3,4);
   }
 
   /* getter/setter functions for convenient python-side members

